frontend/src/pages/RoadDetails.tsx

```diff
@@ -62,26 +61,29 @@ const RoadDetails = () => {
   }, []);
 
   return (
-    <div>
-      <TopBar isToggleOn={setShowRoadImageMode} />
+    <div className="div-mapAndRoad-container">
```

Comment:
is the class div-mapAndRoad-container exists in the css files?

Reply:
I added it only so when I did the inspect on the page I could recognise the div

Reply:
should I remove the className?